src/Microsoft.Azure.SignalR.Management/RestHubLifetimeManager.cs

```diff
@@ -351,6 +358,67 @@ public async Task SendStreamCompletionAsync(string connectionId, string streamId
         await _restClient.SendWithRetryAsync(api, HttpMethod.Post, cancellationToken: cancellationToken);
     }
 
+#if NET7_0_OR_GREATER
+#nullable enable
+    public override async Task<T> InvokeConnectionAsync<T>(string connectionId, string methodName, object?[] args, CancellationToken cancellationToken = default)
+    {
+        // Validate input parameters
+        if (string.IsNullOrEmpty(methodName))
+        {
+            throw new ArgumentException(NullOrEmptyStringErrorMessage, nameof(methodName));
+        }
+        if (string.IsNullOrEmpty(connectionId))
+        {
+            throw new ArgumentException(NullOrEmptyStringErrorMessage, nameof(connectionId));
+        }
+
+        // Get API endpoint and prepare for the request
+        var api = _restApiProvider.SendClientInvocation(_appName, _hubName, connectionId);
+        string? responseContent = null;
+        var isSuccessStatusCode = false;
+        // Send request and capture the response
+        await _restClient.SendMessageWithRetryAsync(
+            api,
+            HttpMethod.Post,
+            methodName,
+            args,
+            async response =>
+            {
+                responseContent = await response.Content.ReadAsStringAsync();
+                isSuccessStatusCode = response.IsSuccessStatusCode;
+                return response.IsSuccessStatusCode || response.StatusCode == HttpStatusCode.BadRequest;
+            },
+            cancellationToken: cancellationToken);
+
+        // Ensure we have a response
+        if (string.IsNullOrWhiteSpace(responseContent))
+        {
+            throw new HubException("Response content is null or empty");
+        }
+
+        if (!isSuccessStatusCode)
+        {
+            throw new HubException(responseContent);
+        }
+
+        var root = JsonNode.Parse(responseContent)
```

Comment:
We should respect the JSON deserialization settings in `ServiceManagerOptions.ObjectSerializer`.